### ISSUE TITLE
.

### ISSUE THREAD

**Issue by LSKhappychild** [vacancy/Synchronized-BatchNorm-PyTorch#49]
Sry confused wrong repo
